File: ExamSolution/Examshubham94/JavaExam/src/PartTimeStaff.java
package payroll;
public class PartTimeStaff extends Staff{

	private int hours;

	public PartTimeStaff(String name,int address, int hours){
		super(name, address);
		this.hours = hours;
	
	}

	public double getSalery(){
	
	return hours<180?200:100;

	}

}

File: ExamSolution/93 Shreyash/Dotnet/Staff/Met/Payroll/PartTimeStaff.cs
namespace Met.Payroll;

public class PartTimeStaff : Staff
{
    public int NumberOfHours { get; set; }
    public double RatePerHour { get; set; }

    public PartTimeStaff(string name, string address, int hours = 180, double rate = 200)
    {
        Name = name;
        Address = address;
        NumberOfHours = hours;
        RatePerHour = rate;
    }

}